From be57dfe3bd539c5721207e9adcbd271b02cc060a Mon Sep 17 00:00:00 2001
From: Eyal Reizer <eyalr@ti.com>
Date: Mon, 15 Oct 2012 11:44:36 +0530
Subject: [PATCH 14/23] da830_omapl137_defconfig: activate wlan related
 switches

Activate the switches needed for working with a wl18xx wifi module
integrated with the da830-wifi board

Signed-off-by: Eyal Reizer <eyalr@ti.com>
---
 arch/arm/configs/da830_omapl137_wifi_defconfig |   42 +++++++++++++++---------
 1 file changed, 27 insertions(+), 15 deletions(-)

diff --git a/arch/arm/configs/da830_omapl137_wifi_defconfig b/arch/arm/configs/da830_omapl137_wifi_defconfig
index 964d741..b0cc7eb 100644
--- a/arch/arm/configs/da830_omapl137_wifi_defconfig
+++ b/arch/arm/configs/da830_omapl137_wifi_defconfig
@@ -296,6 +296,7 @@ CONFIG_DA830_WIFI_EVM=y
 # CONFIG_DA830 is not set
 CONFIG_DA830_UI_LCD=y
 # CONFIG_DA830_UI_NAND is not set
+CONFIG_DA830_WL18XX=y
 CONFIG_GPIO_PCA953X=y
 # CONFIG_KEYBOARD_GPIO_POLLED is not set
 CONFIG_DAVINCI_MUX=y
@@ -513,16 +514,23 @@ CONFIG_NETFILTER_ADVANCED=y
 # CONFIG_NETFILTER_NETLINK_ACCT is not set
 # CONFIG_NETFILTER_NETLINK_QUEUE is not set
 # CONFIG_NETFILTER_NETLINK_LOG is not set
-# CONFIG_NF_CONNTRACK is not set
-# CONFIG_NETFILTER_XTABLES is not set
+CONFIG_NF_CONNTRACK=y
+CONFIG_NF_CONNTRACK_PROCFS=y
+CONFIG_NETFILTER_XTABLES=y
 # CONFIG_IP_VS is not set
 
 #
 # IP: Netfilter Configuration
 #
-# CONFIG_NF_DEFRAG_IPV4 is not set
+CONFIG_NF_DEFRAG_IPV4=y
+CONFIG_NF_CONNTRACK_IPV4=y
 # CONFIG_IP_NF_QUEUE is not set
-# CONFIG_IP_NF_IPTABLES is not set
+CONFIG_IP_NF_IPTABLES=y
+CONFIG_IP_NF_FILTER=y
+CONFIG_IP_NF_TARGET_LOG=y
+CONFIG_NF_NAT=y
+CONFIG_NF_NAT_NEEDED=y
+CONFIG_IP_NF_TARGET_MASQUERADE=y
 # CONFIG_IP_NF_ARPTABLES is not set
 
 #
@@ -531,6 +539,7 @@ CONFIG_NETFILTER_ADVANCED=y
 # CONFIG_NF_DEFRAG_IPV6 is not set
 # CONFIG_IP6_NF_QUEUE is not set
 # CONFIG_IP6_NF_IPTABLES is not set
+
 # CONFIG_IP_DCCP is not set
 # CONFIG_IP_SCTP is not set
 # CONFIG_RDS is not set
@@ -566,6 +575,7 @@ CONFIG_BQL=y
 # CONFIG_BT is not set
 # CONFIG_AF_RXRPC is not set
 CONFIG_WIRELESS=y
+CONFIG_WIRELESS_EXT=y
 # CONFIG_CFG80211 is not set
 # CONFIG_LIB80211 is not set
 
@@ -573,7 +583,7 @@ CONFIG_WIRELESS=y
 # CFG80211 needs to be enabled for MAC80211
 #
 # CONFIG_WIMAX is not set
-# CONFIG_RFKILL is not set
+CONFIG_RFKILL=y
 # CONFIG_NET_9P is not set
 # CONFIG_CAIF is not set
 # CONFIG_CEPH_LIB is not set
@@ -590,7 +600,8 @@ CONFIG_UEVENT_HELPER_PATH="/sbin/hotplug"
 # CONFIG_DEVTMPFS is not set
 CONFIG_STANDALONE=y
 CONFIG_PREVENT_FIRMWARE_BUILD=y
-# CONFIG_FW_LOADER is not set
+CONFIG_FW_LOADER=y
+CONFIG_FIRMWARE_IN_KERNEL=y
 # CONFIG_DEBUG_DRIVER is not set
 # CONFIG_DEBUG_DEVRES is not set
 # CONFIG_SYS_HYPERVISOR is not set
@@ -958,7 +969,8 @@ CONFIG_TOUCHSCREEN_TSC2004=y
 # CONFIG_TOUCHSCREEN_W90X900 is not set
 # CONFIG_TOUCHSCREEN_ST1232 is not set
 # CONFIG_TOUCHSCREEN_TPS6507X is not set
-# CONFIG_INPUT_MISC is not set
+CONFIG_INPUT_MISC=y
+CONFIG_INPUT_UINPUT=y
 
 #
 # Hardware I/O ports
@@ -2068,9 +2080,9 @@ CONFIG_HAVE_ARCH_KGDB=y
 #
 # Security options
 #
-# CONFIG_KEYS is not set
 # CONFIG_SECURITY_DMESG_RESTRICT is not set
-# CONFIG_SECURITY is not set
+CONFIG_KEYS=y
+CONFIG_SECURITY=y
 # CONFIG_SECURITYFS is not set
 CONFIG_DEFAULT_SECURITY_DAC=y
 CONFIG_DEFAULT_SECURITY=""
@@ -2101,7 +2113,7 @@ CONFIG_CRYPTO=y
 # CONFIG_CRYPTO_CBC is not set
 # CONFIG_CRYPTO_CTR is not set
 # CONFIG_CRYPTO_CTS is not set
-# CONFIG_CRYPTO_ECB is not set
+CONFIG_CRYPTO_ECB=y
 # CONFIG_CRYPTO_LRW is not set
 # CONFIG_CRYPTO_PCBC is not set
 # CONFIG_CRYPTO_XTS is not set
@@ -2120,7 +2132,7 @@ CONFIG_CRYPTO=y
 # CONFIG_CRYPTO_GHASH is not set
 # CONFIG_CRYPTO_MD4 is not set
 # CONFIG_CRYPTO_MD5 is not set
-# CONFIG_CRYPTO_MICHAEL_MIC is not set
+CONFIG_CRYPTO_MICHAEL_MIC=y
 # CONFIG_CRYPTO_RMD128 is not set
 # CONFIG_CRYPTO_RMD160 is not set
 # CONFIG_CRYPTO_RMD256 is not set
@@ -2134,9 +2146,9 @@ CONFIG_CRYPTO=y
 #
 # Ciphers
 #
-# CONFIG_CRYPTO_AES is not set
+CONFIG_CRYPTO_AES=y
 # CONFIG_CRYPTO_ANUBIS is not set
-# CONFIG_CRYPTO_ARC4 is not set
+CONFIG_CRYPTO_ARC4=y
 # CONFIG_CRYPTO_BLOWFISH is not set
 # CONFIG_CRYPTO_CAMELLIA is not set
 # CONFIG_CRYPTO_CAST5 is not set
@@ -2176,7 +2188,7 @@ CONFIG_CRC_CCITT=m
 CONFIG_CRC_T10DIF=m
 # CONFIG_CRC_ITU_T is not set
 CONFIG_CRC32=y
-# CONFIG_CRC7 is not set
+CONFIG_CRC7=y
 # CONFIG_LIBCRC32C is not set
 # CONFIG_CRC8 is not set
 CONFIG_ZLIB_INFLATE=y
@@ -2193,5 +2205,5 @@ CONFIG_HAS_DMA=y
 CONFIG_DQL=y
 CONFIG_NLATTR=y
 CONFIG_GENERIC_ATOMIC64=y
-# CONFIG_AVERAGE is not set
+CONFIG_AVERAGE=y 
 # CONFIG_CORDIC is not set
-- 
1.7.9.5

